content/rma_intro.tex

```diff
@@ -16,31 +16,65 @@
 one-sided communication interfaces via \Celev{} generic selection 
 (\Celev{} \S6.5.1.1\footnote{Formally, the \Celev{} specification is ISO/IEC 9899:2011(E).})
 for block, scalar, and block-strided put and get communication. 
-Such type-generic routines are supported for the ``standard \ac{RMA} types’’
-identified in Table \ref{stdrmatypes}.
+Such type-generic routines are supported for the
+\newtext{\Celev{}
+``standard integer types'' (\Celev{} \S6.2.5.7),
+``real floating types'' (\Celev{} \S6.2.5.10),
+``exact-width integer types'' (\Celev{} \S7.20.1.1),
+``integer types capable of holding object pointers'' (\Celev{} \S7.20.1.4),
+and the integer types \textit{size\_t} and \textit{ptrdiff\_t} defined in
+stddef.h (\Celev{} \S7.19), which are collectively called the}
+``standard \ac{RMA} types''
+\newtext{and are listed} \oldtext{identified} in Table \ref{stdrmatypes}.
+\newtext{Implementations may optionally support additional types.}
+
+\textbf{\newtext{Note to implementors:}}
+\begin{itemize}
+\item \newtext{%
+  In order to provide compatibility between C and C++, implementors may elect
+  to include a dependence on stdbool.h, which provides the macro \textit{bool}.
```

Comment:
Right, but I can also have a C++ case in my header file that uses the ```bool``` type instead of ```_Bool```.  What is a bit concerning, is that if the C/C++ objects are not compatible then I need to have a proper C++ binding to support this.